### PR TITLE
Fix transfer certificate error

### PR DESCRIPTION
**Pull request by soanvig**
Fixes issue of race condition between issuer-api certificate persistance and returning result of issuance from batchIssue command

### CODE REVIEW

File: packages/origin-247-transfer/src/utils/parseRedisUrl.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ * therefore https://github.com/dabroek/node-cache-manager-ioredis/pull/1
+ * and https://github.com/luin/ioredis/blob/4680211fe853831f9ff3a3eb69f16d5db6bfbabd/lib/utils/index.ts
+ */
+
+function isInt(value): value is string {
+    const x = parseFloat(value);
+    return !isNaN(value) && (x | 0) === x;
+}
+
+export function parseRedisUrl(url) {
```
Comment (soanvig) [inline packages/origin-247-transfer/src/utils/parseRedisUrl.ts:16]: it's not my function, see comment

Comment (JosephBagaric) [reply packages/origin-247-transfer/src/utils/parseRedisUrl.ts:16]: It's pasted from here? https://github.com/luin/ioredis/blob/4680211fe853831f9ff3a3eb69f16d5db6bfbabd/lib/utils/index.ts#L243-L292

Could you then put this link in the comment above just to give attributions? Thanks.

Comment (JosephBagaric) [reply packages/origin-247-transfer/src/utils/parseRedisUrl.ts:16]: It's a link to the file, not the exact function. And it's not above that function but the whole file + it doesn't explicitly say it's copied.

Comment (soanvig) [reply packages/origin-247-transfer/src/utils/parseRedisUrl.ts:16]: Okey, gonna fix that

**Pull request merged**